plugins/events-backend/src/service/EventsBackend.ts

```diff
@@ -26,6 +26,7 @@ import { DefaultEventBroker } from './DefaultEventBroker';
  * A builder that helps wire up all component parts of the event management.
  *
  * @public
+ * @deprecated `EventBroker`, `EventPublisher`, and `EventSubscriber` got replaced by `EventService` and its methods.
```

Comment:
is the backend itself really deprecated?

Reply:
Yes, the backend itself is not needed anymore. You can subscribe directly using the event service and mostly, you would pass the event service as an argument and do the pub-sub internally without exposing it at the interface - or at least, it's not required anymore.

It was only used for the legacy backend system, and even there it is not needed anymore.

Reply:
Interesting. I spontaneously thought that that's where the http/webhook ingress implementation lived.

Also I've been thinking in terms of a pg-powered default broker, being driven by a central backend that the distributed brokers post events to, and also likewise subscribe from using for example websockets or long polling or whatever.

Maybe that kind of a thing will have to be built now as a completely standalone service, outside of the events impl itself.

Reply:
The HTTP ingress is part of the events-backend package but not part of the `EventsBackend` class. It was only used to wire up stuff incl. adding publishers and subscribers. The HTTP ingress itself was also added as a publisher.